### PR TITLE
Add description for Create a Derived Table

### PR DESCRIPTION
**Pull request by MrAlecJohnson**
On the tools index page, add a description and link for Create a Derived Table.

### CODE REVIEW

File: source/documentation/tools/index.md [hunk 1]
```diff
@@ -18,8 +18,9 @@ The data engineering team maintain a number of databases on the Analytical Platf
 
 ### Data Uploader
 Under construction
+
 ### Create a Derived Table
-Under construction
+Implements a tool called dbt, for creating persistent derived tables in Athena. Visit the [Create a Derived Table repo](https://github.com/moj-analytical-services/create-a-derived-table) to see progress, or sign up for testing on the #ask-data-modelling Slack channel.
```
Comment (nicholsondarius) [inline source/documentation/tools/index.md:23]:
```suggestion
Implements a tool called dbt, for creating persistent derived tables in Athena. Visit the [Create a Derived Table repo](https://github.com/moj-analytical-services/create-a-derived-table) to see progress, or sign up for testing on the [#ask-data-modelling](https://asdslack.slack.com/archives/C03J21VFHQ9) Slack channel.
```

Comment (nicholsondarius) [reply source/documentation/tools/index.md:23]: That URL will remain the same even if the channel name changes.

**Review by nicholsondarius — COMMENTED** [review]
Just adding a link to the Slack channel.

File: source/documentation/tools/index.md (hunk 1 above)
Comment (MrAlecJohnson) [reply source/documentation/tools/index.md:23]: Fantastic, didn't know we could do that - thanks

**Review by nicholsondarius — APPROVED** [review]

**Pull request merged**